clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/narrowing-conversions.rst

```diff
@@ -27,6 +27,29 @@ This check will flag:
  - All applications of binary operators with a narrowing conversions.
    For example: ``int i; i+= 0.1;``.
 
+Note that arithmetic with integer types may perform implicit conversions if the used integer types are smaller than ``int``.
```

Comment:
wrap all lines on 80 column

Reply:
the warning message and `static_cast` in the code example are still over 80. Wrapping would hinder readability in the example in my opinion and I believe this (was) is not commonly done in code. Otherwise done.